index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Felipe Oyarzo">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@400;500;700&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="./assets/css/style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <a href="#"><img src="./assets/img/ic_menu.png" alt="Hamburguer menu icon"></a>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <div class="text-wrapper">
+                <h1>Felipe<br>Oyarzo</h1>
+                <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of
+                    my works. If you like what you see and have a project you need coded, don't hesitate and contact me.
+                </p>
+            </div>
+            <button class="button" type="button" id="btnHeadline">Start collaboration</button>
+            <div class="geometric"></div>
+            <!-- <img src="./assets/img/image_geometry_2.png" alt="Geometric shapes" class="geometric"> -->
```

Comment:
- [ ] [optional] Kindly ensure you don't have commented code in your codebase, its not good practise

Reply:
Thanks for your feedback @danmainah , I totally forgot to delete that commented code, I appreciate it :)